include/mbedtls/cmac.h

```diff
@@ -35,104 +36,118 @@ extern "C" {
 #define MBEDTLS_DES3_BLOCK_SIZE         8
 
 #if defined(MBEDTLS_AES_C)
-#define MBEDTLS_CIPHER_BLKSIZE_MAX      16  /* longest used by CMAC is AES */
+#define MBEDTLS_CIPHER_BLKSIZE_MAX      16  /* The longest block used by CMAC is AES. */
```

Comment:
Suggestion: "The longest block-size used by CMAC is that of AES."

Reply:
fixed.